source/documentation/appendix-docs/conda.md

```diff
@@ -1,89 +1,73 @@
-# Conda Package Management
+# Package management
 
-## Unified package management for both R and Python
+## Introduction
 
-### Faster builds
+A key element of ensuring analysis is reproducible is maintaining a record of the versions of packages used to produce said analysis. There are a number of package management tools available for both R and Python. Within the MoJ we recommend primarily using [Conda](#Conda) for both languages.
 
-CRAN mirrors do not provide linux compiled binaries
-for packages. This means a long wait when doing `install.packages` both in an R
-Studio session and when running a Docker build for a `shiny` application.
+## R
 
-### Cross Language
+### Packrat
 
-Both `dbtools` and `s3tools` rely on Python packages through the `reticulate` R
-to Python bridge. `packrat` only handles R dependencies, this means that
-`packrat` is not enough to reproducibly and reliably manage all of your
-application's dependencies.
+Packrat is the most well-known package management tool for R. There's more information about it here: https://rstudio.github.io/packrat/
 
-`conda` supports managing both Python and R dependencies in a single
-`environment`. It can make sure all of these libraries are compatible with each
-other.
+It has some significant downsides. It can be quite temperamental, and difficult to debug when things go wrong - in the earlier days of the Analytical Platform, the majority of support issues related to getting Packrat working. Furthermore, CRAN mirrors do not provide linux compiled binaries for packages. This means a long wait when doing `install.packages` both in an RStudio session and when running a Docker build for a `shiny` application.
```

Comment:
"CRAN mirrors do not provide Linux compiled binaries for packages" - this is a pretty technical statement, and I would guess it will be baffling to newcomers. You can leave it in, but maybe add something like "this means that the packages have to be compiled on the Analytical Platform, which means a long wait etc".

Reply:
Clarified this